json-definitions/v3/tech-record/get/psv/complete/index.json

```diff
@@ -85,9 +85,6 @@
       "anyOf": [
```

Comment:
you'll have to remove the `anyOf` if you only have one option in the array, or it'll fail validation

Reply:
Cheers, working out what the hell is wrong with this PR atm tho, hate your bot (which is my fault)

Reply:
sorry about the bot 😢